Backend/src/main/resources/application-mysql-dev-example.properties

```diff
@@ -1,7 +1,7 @@
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.datasource.url= jdbc:mysql://localhost:3306/seproject?useSSL=false&serverTimezone=UTC&useLegacyDatetimeCode=false&createDatabaseIfNotExist=true&allowPublicKeyRetrieval=true
 spring.datasource.username= pw2020
-spring.datasource.password= MysqL12!@
+spring.datasource.password= roTaD6fo
```

Comment:
Don't change this file

Reply:
My bad. Didn't mean to commit this change.